src/js/select2/core.js

```diff
@@ -335,7 +335,7 @@ define([
 
           evt.preventDefault();
         } else if (key === KEYS.ENTER) {
-          self.trigger('results:select', {});
+          self.trigger('results:select', evt);
```

Comment:
This should be wrapped as `originalEvent` similar to how we do it for other events.

Reply:
Ah, okay.  I wrapped it like that in `results.js` but thought it would be better to be consistent here with the call to `select.close` 4 lines above this, and to be consistent with the other handlers in `results.js`  (like the `mouseup` handler on line 448).   

Otherwise the `results:select` handler will have two `originalEvent: **evt**.originalEvent` wrappings which are inconsistent with the existing wrapping in the `mouseup `handler.  Seems like trading one inconsistency for two others.

Happy to make the change as long as you're okay with that.